Core/include/Utils/IDUtils.h

```diff
@@ -1,6 +1,5 @@
 #pragma once
 
-#include "../../External/uuid/uuid_v4.h"
 #include "../../External/uuid/uuid_v4.h"
 
-UUIDv4::UUIDGenerator<std::mt19937_64> uuidGen;
\ No newline at end of file
+extern UUIDv4::UUIDGenerator<std::mt19937_64> uuidGen;
```

Comment:
Add comment explaining what `extern` is and why we should be conscious of adding it when needed.

Reply:
I dont like adding verbose comments to code, people should be googling extern and its use.

Reply:
Understandable